File: Programming EF Code First/chapter3/Chapter3/BAGA_DataAnnotations/Model/Destination.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Model
{
  public class Destination
  {
    public int DestinationId { get; set; }
    [Required]
    public string Name { get; set; }
    public string Country { get; set; }
    [MaxLength(500)]
    public string Description { get; set; }
    [Column(TypeName = "image")]
    public byte[] Photo { get; set; }

    public List<Lodging> Lodgings { get; set; }
  }
}

File: Programming EF Code First/chapter3/Chapter 3 VB/Data Annotations/Model/Destination.vb
Imports System.ComponentModel.DataAnnotations

Namespace Model
  Public Class Destination
	Public Property DestinationId() As Integer
	<Required>
	Public Property Name() As String
	Public Property Country() As String
	<MaxLength(500)>
	Public Property Description() As String
	<Column(TypeName := "image")>
	Public Property Photo() As Byte()

	Public Property Lodgings() As List(Of Lodging)
  End Class
End Namespace